setup.cfg

```diff
@@ -75,15 +75,15 @@ markers =
 [isort]
 multi_line_output = 3
 line_length = 88
-known_third_party = appdirs, astroid, sphinx, isort, pytest, mccabe, six, toml
+known_third_party = platformdirs, astroid, sphinx, isort, pytest, mccabe, six, toml
 include_trailing_comma = True
 skip_glob = tests/functional/**,tests/input/**,tests/extensions/data/**,tests/regrtest_data/**,tests/data/**,astroid/**,venv/**
 src_paths = pylint
 
 [mypy]
 scripts_are_modules = True
 
-[mypy-appdirs]
+[mypy-platformdirs]
```

Comment:
@Pierre-Sassoulas FYI, you should be able to remove this, since another benefit of `platformdirs` is it is fully statically typed.

Reply:
@CAM-Gerlach Thanks for the heads up! We can indeed remove it (#4889), although we need to bump the required version first (#4890).